proxy/http/unit-tests/test_MemView.cc

```diff
@@ -0,0 +1,24 @@
+/** @file
+
+  Include a needed TS lib source file to avoid mocking.
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#include <ts/MemView.cc>
```

Comment:
Where are the tests in this file?

Reply:
There is no test code in test_MemView.cc .  It's linked in to test_ForwardedConfig.  It was easier to use the actual implementation of ts::StringView in the unit testing rather than mocking it.